### PR TITLE
Rf06 tc 027 1

### PR DESCRIPTION
**Pull request by borisytu2014**
imagen
lista contratos tc

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class ListContract extends BasePage {
+
+    @FindBy(id = "ContractTable")
+    private WebElement paginElement;
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java:11]: Please change the name of the webElement

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.managepage.Contract.ListContract;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class SubMenuPersonalContract extends BasePage {
+
+    @FindBy(id = "smContract")
+    private WebElement usuarioSubMenu;
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java:12]: Change the name for the WebElement
Don't use Spanish with English

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
     @FindBy(id = "menuPersonal")
     private WebElement menuPersonal;
 
-    @FindBy(id = "incident-lnk")
+    @FindBy(id= "incident-lnk")
```
Comment (ctola91) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java:16]: This change is not necessary for your feature, please remove this change.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    public boolean isContractListVisible() {
```
Comment (jdpapiero) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Contract/ListContract.java:13]: change name method. Ej.: validateContractListIsVisible()

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java (hunk 2 above)
Comment (jdpapiero) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/SubMenuPersonalContract.java:12]: Ej.: subMenuUsuario

File: src/test/resources/features/users/listUsers.feature [hunk 5]
```diff
@@ -9,4 +9,4 @@ Feature: List User
     Given 'Menu principal' estee cargado
     And seleccionar menu 'Personal' en la pagina 'Menu Principal'
     And Seleccionar submenu 'Usuario' en menu 'Personal'
```
Comment (jdpapiero) [inline src/test/resources/features/users/listUsers.feature:11]: first letter in step should be lowercase

File: src/test/resources/features/users/listUsers.feature [hunk 6]
```diff
@@ -9,4 +9,4 @@ Feature: List User
...
-    Then Validar que la 'Lista de Usuarios' este visible
+    Then Validar que la 'Lista de Usuarios' este visible
```
Comment (jdpapiero) [inline src/test/resources/features/users/listUsers.feature:12]: first letter in step should be lowercase

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 7]
```diff
@@ -110,4 +114,24 @@ public void seleccionarSubMenuEquipamiento() throws Throwable {
     public void validarListaDeEquipamientos() throws Throwable {
         Assert.assertTrue(listEquipment.isEquipmentListVisible(), String.format(ErrorMessage.ERROR_MESSAGE_ELEMENT_VISIBLE, "Equipments title"));
     }
+
+    @Given("^menu principal este cargado en pagina de inicio$")
+    public void seleccionarPaginaInicio() throws Throwable {
+        menu = home.getHomeMenu();
+    }
+
+    @And("^seleccionar la opcion 'Personal' en la pagina 'menu principal'$")
```
Comment (marbusjim) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java:123]: I think you should change your step definition, example: "Seleccionar la opcion 'personal' de menu en la pagina 'Home'"

**Pull request closed without merging**